cumulus_library/studies/core/encounter.sql

```diff
@@ -68,8 +68,7 @@ WITH powerset AS (
         ce.age_at_visit,
         cp.gender,
         cp.race_display,
-        cp.ethnicity_display,
-        cp.postalcode3
+        cp.ethnicity_display
```

Comment:
Just confirming this was intentional

Reply:
yep, sorry -  it was part of the request on #38